src/foundry/client/apps/form.d.mts

```diff
@@ -59,11 +59,13 @@ declare global {
   > extends Application<Options> {
     /**
      * @param object  - Some object or entity which is the target to be updated.
+     *                  (default: `{}`)
+     *
      * @param options - Additional options which modify the rendering of the sheet.
      *                  (default: `{}`)
      * @remarks Foundry allows passing no value to the constructor at all.
      */
-    constructor(object: ConcreteObject, options?: Partial<Options>);
+    constructor(object?: ConcreteObject, options?: Partial<Options>);
```

Comment:
So we can't make this blanket optional. Could you open an issue for this?

The issue is basically if `ConcreteObject` contains required properties. You can see the attempt to make this work in the overload right below.